docs/runtime-contract.md

```diff
@@ -423,27 +392,23 @@ Platform providers MAY provide additional platform-specific mount points
 the location and contents of the mount points SHOULD be documented by the
 platform provider.
 
-### Namespaces
-
-The namespace configuration MUST be provided by the operator or platform
-provider; developers or container providers MUST NOT set or assume a particular
-namespace configuration.
```

Comment:
Why remove this?

Reply:
Looking at this again I am going to put it back and update it. I was assuming Kubernetes Namespace and not Container Namespace when I was deleting this.